packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/modelservice/ModelServiceProxy.java

```diff
@@ -0,0 +1,58 @@
+package software.uncharted.terarium.hmiserver.proxies.modelservice;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import software.uncharted.terarium.hmiserver.models.modelservice.Graph;
+import software.uncharted.terarium.hmiserver.models.modelservice.SimulateParams;
+import software.uncharted.terarium.hmiserver.models.modelservice.ModelCompositionParams;
+
+@RegisterRestClient(configKey = "model-service")
+@Produces(MediaType.APPLICATION_JSON)
+public interface ModelServiceProxy {
+	@PUT
+	@Path("api/models")
```

Comment:
This could have `@Path("/api/models")` declared for the entire Interface, with each methods' paths becoming just `@Path("/{modelId"}`, etc.  This would closer follow the convention used in the other proxy interfaces.